resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
+	"offset": 0 // Delay of this tag between event happening and being sent to server - milliseconds
+  },
+
+  "event": {
+	"page_id": "", // ID of the page on which this event happened.
+	"parent_id": "", // Parent event ID if this is a child event.
```

Comment:
Do you mean in the client?

In Andrew's example:
```
{
  collations: [
    "videoclip:12345",
    "o-video-instance:0000-0000-000-0000",
    "origami:o-video"
  ]
}
```

How do I programatically know which order these collation id's should go in? e.g. is instance a parent of videoclip and videoclip is a child of o-video? Or is it another way around?
I'd have to keep a map or something like that for each type of component/event.